src/RedBlackBinaryTree.sol

```diff
@@ -23,7 +23,8 @@ library RedBlackBinaryTree {
         mapping(address => uint256) keyToValue; // Maps key to its value.
     }
 
-    /** @dev Returns the smallest value in the tree `_self`.
+    /**
+     * @dev Returns the smallest value in the tree `_self`.
      *  @param _self The tree to search in.
      */
```

Comment:
We changed the style of these right ?

Reply:
It will be tackle by this issue. https://github.com/morpho-dao/morpho-data-structures/issues/125